src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsExpensesFromPage.tsx

```diff
@@ -129,32 +130,37 @@ function WorkspaceWorkflowsApprovalsExpensesFromPage({policy, isLoadingReportDat
         const members: Member[] = selectedMembers.map((member) => ({displayName: member.text, avatar: member.icons?.[0]?.source, email: member.login}));
         Workflow.setApprovalWorkflowMembers(members);
 
-        const firstApprover = approvalWorkflow?.approvers?.[0]?.email;
-        if (approvalWorkflow?.isBeingEdited && firstApprover) {
-            Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_EDIT.getRoute(route.params.policyID, firstApprover));
-        } else {
+        if (route.params.backTo) {
+            Navigation.navigate(route.params.backTo);
+            return;
+        }
+
+        if (approvalWorkflow?.flow === 'create') {
             Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_APPROVER.getRoute(route.params.policyID, 0));
+        } else {
```

Comment:
Is this else block used for editing workflow flow?

Reply:
Yes 👍